src/main/java/spireMapOverhaul/zones/gremlinTown/events/GremlinPerformance.java

```diff
@@ -0,0 +1,173 @@
+    public static final String ID = makeID(GremlinPerformance.class.getSimpleName());
+    private static final EventStrings eventStrings;
+    private static final String[] DESCRIPTIONS;
+    private static final String NAME;
+    private static final String IMAGE_PATH;
+    private static final String[] OPTIONS;
+
+    private static final int REMOVAL_PRICE = 50;
+    private static final int REMOVAL_PRICE_A15 = 75;
+    private static final int MAX_HP_PRICE = 125;
+    private static final int MAX_HP_PRICE_A15 =150;
+    private static final int UPGRADE_PRICE = 250;
+    private static final int UPGRADE_PRICE_A15 = 300;
+
+    private static final int MAX_HP_GAIN = 10;
+
+    private CUR_SCREEN screen;
+
+    static {
+        eventStrings = CardCrawlGame.languagePack.getEventString(ID);
+        NAME = eventStrings.NAME;
+        DESCRIPTIONS = eventStrings.DESCRIPTIONS;
+        OPTIONS = eventStrings.OPTIONS;
+        IMAGE_PATH = makeImagePath("events/GremlinTown/" + GremlinPerformance.class.getSimpleName() + ".jpg");
+    }
+
+    public static boolean bonusCondition() {
+        if (adp() == null)
+            return false;
+        return adp().gold >= getMaxHpPrice();
+    }
+
+    public GremlinPerformance() {
+        super(NAME, DESCRIPTIONS[0], IMAGE_PATH);
+        screen = CUR_SCREEN.INTRO1;
+        imageEventText.updateBodyText(DESCRIPTIONS[0]);
+        imageEventText.setDialogOption(OPTIONS[0]);
+    }
+
+    @Override
+    public void onEnterRoom() {
+        CardCrawlGame.music.precacheTempBgm("BOSS_CITY");
+    }
+
+    public void update() {
+        super.update();
+        if (!AbstractDungeon.gridSelectScreen.selectedCards.isEmpty()) {
+            AbstractCard c = AbstractDungeon.gridSelectScreen.selectedCards.get(0);
+            AbstractDungeon.topLevelEffects.add(new PurgeCardEffect(c, (float)(Settings.WIDTH / 2), (float)(Settings.HEIGHT / 2)));
+            adp().masterDeck.removeCard(c);
+            AbstractDungeon.gridSelectScreen.selectedCards.remove(c);
+        }
+    }
+
+    protected void buttonEffect(int buttonPressed) {
+        if (screen == CUR_SCREEN.INTRO1) {
+            screen = CUR_SCREEN.INTRO2;
+            imageEventText.clearAllDialogs();
+            imageEventText.setDialogOption(OPTIONS[0]);
+            imageEventText.updateBodyText(DESCRIPTIONS[1]);
+            CardCrawlGame.music.unsilenceBGM();
+            AbstractDungeon.scene.fadeOutAmbiance();
+            CardCrawlGame.music.playPrecachedTempBgm();
+        } else if (screen == CUR_SCREEN.INTRO2) {
+            screen = CUR_SCREEN.DECISION;
+            imageEventText.clearAllDialogs();
+            imageEventText.setDialogOption(OPTIONS[1].replace("{0}", String.valueOf(getRemovalPrice())));
+            imageEventText.setDialogOption(OPTIONS[2].replace("{0}", String.valueOf(getMaxHpPrice())));
+            imageEventText.setDialogOption(OPTIONS[2].replace("{1}", String.valueOf(MAX_HP_GAIN)));
```

Comment:
You have the same option repeated here, with different partial text replacements. I think what you want is the following single line instead of these two lines:
```
imageEventText.setDialogOption(OPTIONS[2].replace("{0}", String.valueOf(getMaxHpPrice())).replace("{1}", String.valueOf(MAX_HP_GAIN)));
```